src/main/java/com/google/sps/data/DatastoreScheduledInterviewDao.java

```diff
@@ -68,7 +70,10 @@ public DatastoreScheduledInterviewDao() {
         new FilterPredicate("interviewee", FilterOperator.EQUAL, email);
     CompositeFilter compositeFilter =
         CompositeFilterOperator.or(interviewerFilter, intervieweeFilter);
-    Query query = new Query("ScheduledInterview").setFilter(compositeFilter);
+    Query query =
```

Comment:
You should add the fact that this is sorted to the comment at the top of the method.